Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -1201,6 +1201,7 @@ def test_is_demisto_version_build_ge(mocker):
     assert not is_demisto_version_ge('6.0.0', '50001')
     assert not is_demisto_version_ge('6.1.0', '49999')
     assert not is_demisto_version_ge('5.5.0', '50001')
+    assert is_demisto_version_ge('6.0.0', '6')
```

Comment:
Please move to line 1201 to keep consistency.
I would also add a comment explaining this assertion, something like - 'Added with the fix of __link to the issue__ '